build-tools/packages/build-cli/src/commands/generate/typetests.ts

```diff
@@ -0,0 +1,228 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { CliUx, Flags } from "@oclif/core";
+import chalk from "chalk";
+
+import { generateTests, getAndUpdatePackageDetails } from "@fluidframework/build-tools";
+
+import { BaseCommand } from "../../base";
+import { releaseGroupFlag } from "../../flags";
+
+export default class GenerateTypeTestsCommand extends BaseCommand<
+    typeof GenerateTypeTestsCommand.flags
+> {
+    static summary =
+        "Generates type tests based on the individual package settings in package.json.";
+
+    static description = `Generating type tests has two parts: preparing package.json and generating type tests. By default, both steps are run for each package. This can be overridden using the --prepare and --generate flags.`;
```

Comment:
Nit
```suggestion
    static description = `Generating type tests has two parts: preparing package.json and generating test modules. By default, both steps are run for each package. This can be overridden using the --prepare and --generate flags.`;
```